app/src/main/java/cm/aptoide/pt/home/HomeNavigator.java

```diff
@@ -65,10 +65,15 @@ public void navigateWithDownloadUrlAndReward(long appId, String packageName, Str
 
   public void navigateWithAction(HomeEvent click) {
 
+    String tag = click.getBundle()
+        .getTag();
+    if (click.getBundle() instanceof AppBundle) {
+      tag = ((AppBundle) click.getBundle()).getActionTag();
+    }
+
     fragmentNavigator.navigateTo(StoreTabGridRecyclerFragment.newInstance(click.getBundle()
         .getEvent(), click.getType(), click.getBundle()
-        .getTitle(), "default", click.getBundle()
-        .getTag(), StoreContext.home), true);
+        .getTitle(), "default", tag, StoreContext.home), true);
```

Comment:
Looks like we should pass tag and tag action since it's two different things... Because if not, what's the use of having the tag and not tag action in other cases other than AppBundle?